app/templates/my-projects/project/recommendations/add.hbs

```diff
@@ -364,13 +364,142 @@
+                        <span class="display-block">
+                          Votes Against: {{disposition.votingagainstrecommendation}}
+                        </span>
+                        <span class="display-block">
+                          Abstain: {{disposition.votingabstainingonrecommendation}}
+                        </span>
+                        <span class="display-block">
+                          Total Members: {{disposition.totalmembersappointedtotheboard}}
+                        </span>
+                      {{/if}}
+                      <span class="display-block">
+                        Comment: {{disposition.consideration}}
+                      </span>
+                      <hr />
+                    {{/each}}
+                  {{/if}}
+                  {{#if (or (eq this.allActions true) (eq this.allActions false))}}
+                    {{#if (or (eq participantType 'CB') (eq participantType 'BB'))}}
+                      <span class="display-block">
+                        Vote Location: {{this.dispositionForAllActions.votelocation}}
+                      </span>
+                      <span class="display-block">
+                        Vote Date: {{this.dispositionForAllActions.dateofvote}}
+                      </span>
+                    {{/if}}
+                    {{#if (eq this.allActions true)}}
+                      <span class="display-block">
+                        Comment: {{this.dispositionForAllActions.consideration}}
+                      </span>
+                    {{/if}}
+                    <span class="display-block">
+                      Uploaded files:
+                      <ul>
+                        <li><a>Important File <small>(PDF)</small></a></li>
+                        <li><a>Important File <small>(PDF)</small></a></li>
+                        <li><a>Important File <small>(PDF)</small></a></li>
+                      </ul>
+                    </span>
+                  {{/if}}
+                  <p class="callout warning text-small">
+                    Once you submit recommendation information, you can not edit it. You'll need to contact the Lead Planner to make any changes.
+                  </p>
+                  <p>
+                  <button
+                    class="button"
+                    data-test-button="confirmRecommendation"
+                    type="button"
+                    onClick={{action this.submitRecommendations}}
+                  >
+                    Submit Recommendation Information
+                  </button>
+                  <button
+                    class="button clear"
+                    data-test-button="closeModal"
+                    type="button"
+                    onClick={{action this.closeModal}}
+                  >
+                    Cancel <small>(continue editing)</small>
+                  </button>
+                </p>
+              </div>
+            </div>
+          {{/if}}
```

Comment:
Will need to seriously break down the add.hbs template into more components for better maintainability.